ansible/roles/dehydrated/tasks/main.yml

```diff
@@ -86,5 +84,4 @@
   notify:
     # creates:
     # /var/lib/dehydrated/certs/<name>/chain.pem cert.pem privkey.pem fullchain.pem
-    - restart dehydrated
-    - reload nginx
+    - reload nginx and restart dehydrated
```

Comment:
But if you say:
```
    - reload nginx
    - restart dehydrated
```
it doesn't respect the order properly?

Reply:
Nevermind: https://docs.ansible.com/ansible/latest/playbook_guide/playbooks_handlers.html#notifying-handlers

> Handlers are executed in the order they are defined in the handlers section, not in the order listed in the notify statement. Notifying the same handler multiple times will result in executing the handler only once regardless of how many tasks notify it